content/doc/book/managing/system-info.adoc

```diff
@@ -15,13 +17,57 @@ endif::[]
 The *Manage Jenkins >> System Information* page provides detailed information
 about what is available on this Jenkins controller:
 
-* *System Properties* that can be used as arguments
+image::managing/system-info-page.png["System Information page", width=600]
+
+== System properties
+
+Lists system properties that can be used as arguments
 to the command line used to start Jenkins.
-* *Environment Variables* recognized on this system,  with current values.
+
+image::managing/system-properties.png["System properties", width=600]
+
+== Environment variables
+
+*Environment Variables* recognized on this system,  with current values.
 This includes the environment variables defined by Jenkins
 and available on all systems
 as well as environment variables associated with plugins installed on this controller.
-* List of Plugins installed on the system.
-* *Memory Usage* gives a graph that shows the current memory usage for this controller.
 
+image::managing/environment-variables.png["Environment variables", width=600]
+
+== Installed plugins
+
+Provides a comprehensive list of all installed plugins, including their names, versions, and other relevant details.
+
+image::managing/system-plugins.png["Plugins", width=600]
+
+== Memory usage
+
+This section presents a graphical representation of the controller's memory usage, categorized into three timespans for better analysis:
+
+- **Short-term**: Covers memory usage over the last few seconds/minutes for real-time monitoring.
+
+image::managing/memory-usage-short.png["Memory usage short", width=600]
+
+- **Medium-term**: Displays memory trends over the past hour/minutes, useful for detecting gradual memory leaks.
+
+image::managing/memory-usage-medium.png["Memory usage medium", width=600]
+
+- **Long-term**: Illustrates extended memory usage patterns over a day or month to identify recurring issues.
+
+image::managing/memory-usage-long.png["Memory usage long", width=600]
+
+This breakdown allows administrators to monitor memory trends, detect unusual spikes, and optimize resource allocation.
+
+== Thread dump
+
+The **Thread Dump** section captures a real-time snapshot of all active threads running in the Jenkins controller’s JVM. This is essential for diagnosing performance issues, deadlocks, or excessive CPU utilization. Each thread entry includes:
```

Comment:
```suggestion
The **Thread Dump** section provides a link to a page that captures a real-time snapshot of all active threads running in the Jenkins controller’s JVM. This is essential for diagnosing performance issues, deadlocks, or excessive CPU utilization. Each thread entry includes:
```
Suggestion to provide more clarity that the section provides a link to the page with this information.
Additionally, the first screenshot does not need to include the entire page in Jenkins, it can be more focused to the Thread Dump section being described.